### ISSUE TITLE
Accessibility: Color contrast failure on pagination & Add to Cart

### ISSUE THREAD

**Issue by joedolson** [aspirepress/AspireExplorer#35]
The current pagination state is `#6eb9e3`:`#fff`, with a contrast ratio of 2.16:1, well below the minimum contrast ratio of 4.5:1. 

Same color combination is used for the Add to Cart button; because that's a graphical control, it only needs to meet 3:1, but that is also a failure.
